soccer/src/soccer/planning/params.cpp

```diff
@@ -0,0 +1,6 @@
+#include <rj_param_utils/param.hpp>
+
+namespace trajectory_utils {
+    DEFINE_INT64(kMaxIterations, max_iterations, 100, "Limit on iterations")
```

Comment:
I think it should be okay to put these directly in the `trajectory_utils.hpp/cpp` files.